src/pyhf/workspace.py

```diff
@@ -267,10 +267,15 @@ def __init__(self, spec, **config_kwargs):
         """Workspaces hold the model, data and measurements."""
         super(Workspace, self).__init__(spec, channels=spec['channels'])
         self.schema = config_kwargs.pop('schema', 'workspace.json')
-        self.version = config_kwargs.pop('version', None)
+
         # run jsonschema validation of input specification against the (provided) schema
-        log.info("Validating spec against schema: {0:s}".format(self.schema))
-        utils.validate(self, self.schema, version=self.version)
+        log.info(f"Validating spec against schema: {self.schema}")
+        utils.validate(self, self.schema)
+
+        if 'version' in spec:
```

Comment:
this check is not needed since `version` must be in `spec` to pass schema validation.

Reply:
Yes, my bad. I'll make the changes.